### PR TITLE
Code quality

### PR DESCRIPTION
**Pull request by holmesie**
Adjustments to make this repository pass code linting.

Pull requests into js-ecqm-engine require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/TACO-32
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @okeefm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @ljades 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/handler.js [hunk 1]
```diff
@@ -1,13 +1,10 @@
-"use strict" ;
```
Comment (ljades) [inline lib/handler.js:1]: Why are we removing `"use strict" ;`? That's how we enable ES6 functionality, no?

Comment (okeefm) [reply lib/handler.js:1]: The linter says ` 'use strict' is unnecessary inside of modules`

Comment (holmesie) [reply lib/handler.js:1]: Getting a `'use strict' is unnecessary inside of modules` for these from eslint. What exactly does it have to do with enabling ES6 functionality?

Comment (ljades) [reply lib/handler.js:1]: Interesting. Maybe I was only getting that error for specs originally.

**Review by ljades — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
Looks good to me.
